src/main/java/org/ambraproject/wombat/controller/SearchController.java

```diff
@@ -652,59 +645,12 @@ private ModelAndView getFeedModelAndView(Site site, String feedType, String titl
   @RequestMapping(name = "simpleSearch", value = "/search")
   public String search(HttpServletRequest request, Model model, @SiteParam Site site,
                        @RequestParam MultiValueMap<String, String> params) throws IOException {
-    if (!performValidSearch(request, model, site, params, false)) {
+    if (!performValidSearch(request, model, site, params)) {
       return advancedSearchAjax(model, site);
     }
     return site.getKey() + "/ftl/search/searchResults";
   }
 
-  /**
-   * Performs a csv export of a search.
-   *
-   * @param request HttpServletRequest
-   * @param model   model that will contain search results
-   * @param site    site the request originates from
-   * @param params  all URL parameters
-   * @return String indicating template location
-   * @throws IOException
-   */
-
-  @RequestMapping(name = "csvExport", value = "/csvExport", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
```

Comment:
:feelsgood: getting rid of unused features.  I see later that this whole controller is remove, extra nice!

Reply:
I should have mentioned that I talked to Kristen about this & she gave me the go-ahead to get rid of it.